content/developer/reference/frontend/odoo_editor.rst

```diff
@@ -124,8 +124,8 @@ There are three ways to filter commands:
    :ref:`command <reference/frontend/odoo_editor/powerbox/command>`'s
    `isDisabled` entry.
 #. The user can filter commands by simply typing text after opening the
-   Powerbox. It will fuzzy-match that text with the names of the categories and
-   commands.
+   Powerbox. It will fuzzy-match that text with the names, categories and
+   descriptions of the commands.
```

Comment:
Technically the sentence is wrong because it does _not_ fuzzy match in the description.

Reply:
See discussion above.

Reply:
Oh ok I didn't see that you originally did exactly what I thought you should do but that @Zinston had you change it... ^^' I guess it was quite verbose and fuzzy is not technically as incorrect as I thought since, like you say, we still do some conversion work rather than doing a pure exact match anyway, so I guess I overreacted on this one. It's fine by me.